### ISSUE TITLE
[BITV]: On small screens it is not possible to reach a submenu

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/text#5220]
1. Open a text file
2. Set a viewport to the width of 320px
3. Lots of buttons are located inside of the "Remanining actions"
4. Try to select a button which is a `menu` itself like "headings" or "emojis"
5. "undefined" will be written in a text file:
![Screenshot from 2024-01-10 10-22-25](https://github.com/nextcloud/text/assets/6078378/5552f5bd-a0ca-4973-9810-59ece7615200)

**Comment by juliushaertl** [comment]
Is the emoji picker usable anywhere with that screen size?

**Comment by JuliaKirschenheuter** [comment]
@szaimen why it is not usable if scrollbar is there?

**Comment by szaimen** [comment]
> @szaimen why it is not usable if scrollbar is there?

The scrollbar is outside the emojy picker if you look in detail

**Comment by juliushaertl** [comment]
So the menubar in text is sticky, that is why scrolling the page does not move the position of the emoji picker modal. Though I don't see a good way to solve this as of now

**Comment by JuliaKirschenheuter** [comment]
> So the menubar in text is sticky, that is why scrolling the page does not move the position of the emoji picker popover. Though I don't see a good way to solve this as of now

All right @juliushaertl, thank you! We can do what we can for now. @szaimen if you have no ideas how to fix it, we can close the issue.

**Comment by szaimen** [comment]
> @szaimen if you have no ideas how to fix it, we can close the issue.

The only fix I see for now would be disabling the emojiy picker on mobile if the viewport (height or width) is too low for it to work correctly...

**Comment by JuliaKirschenheuter** [comment]
I don't think there is something we can change for now. Thank you @juliushaertl, closing the issue

**Comment by juliushaertl** [comment]
> Emoji picker should be fixed later separately (another issue). Idea from @ShGKme: on small screens a full width of emoji picker can be shown.

Yes, probably also in full height then but we'll need some kind of close button or close on insert. Probably also something to generally think about for the NcPopover component (ref https://github.com/nextcloud-libraries/nextcloud-vue/issues/3159)
